src/software/firmware/srcs/pressure_controller.cpp

```diff
@@ -421,12 +424,18 @@ void PressureController::inhale() {
     m_peakPressure = max(m_pressure, m_peakPressure);
 }
 
-void PressureController::plateau() {
+void PressureController::plateau(uint16_t p_centiSec) {
     // Deviate the air stream outside
     m_blower_valve.close();
 
-    // Close the air stream towards the patient's lungs
-    m_patient_valve.close();
+    // Gently close the air stream towards the patient's lungs
+    if (p_centiSec < (m_plateauStartTime + 10u)) {
+        m_blower_valve.open(((p_centiSec - m_plateauStartTime)
+                             * (VALVE_CLOSED_STATE - m_peakBlowerValveAngle) / 10u)
+                            + m_peakBlowerValveAngle);
+    } else {
+        m_patient_valve.close();
+    }
```

Comment:
This modification has an impact on production version with servo. Are we ok with this modification or the close is now too slow for servo version?

@Benhalor can you check if it's ok?